specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2018-08-01/ResourceHealth.json

```diff
@@ -1095,6 +1128,156 @@
         }
       }
     },
+    "statusBanner": {
+      "description": "banner type of emerging issue",
+      "properties": {
+        "title": {
+          "type": "string",
+          "description": "the banner title."
```

Comment:
It is more common to start with Uppercase in description and end with period ".".

Same for the rest.

This is not consistent within this PR.

Reply:
Good point. I'm changing it. Thanks for the comment.

Reply:
fixed